File: VstCustomer/Models/ORDER.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace VstCustomer
{
    public class ORDERED
    {
        public string ID { get; set; }
        public string EMP_ID { get; set; }
        public DateTime ORDED_DATE { get; set; }
        public string CUSTOMER_ID { get; set; }
        public string ORDER_CR_HR { get; set; }
        public string STS_ST_CLS { get; set; }
        public string STS_ST_SER { get; set; }
        public string SURFACE_CD { get; set; }
        public decimal ORD_THK { get; set; }
        public decimal ORD_WTH { get; set; }
        public string ORD_EDGE { get; set; }
        public decimal ORD_WGT { get; set; }
        public decimal BASE_PRICE { get; set; }
        public decimal EFFECT_PRICE { get; set; }
        public decimal BIDD_PRICE { get; set; }
        public string CONTRACT_NO { get; set; }
        public string ORD_USAGE { get; set; }
        public string ORD_STAT { get; set; }
        public string END_USER { get; set; }
        public int QUANTITY { get; set; }
        public string DELIVERY_TIME { get; set; }
        public string REMARK { get; set; }

        public ORDERED() { }

        public virtual List<ORDERED> Select(string ID = "")
        {
            var sql = "SELECT * FROM ORDERED ";
            if (string.IsNullOrWhiteSpace(ID)) return DBManager<ORDERED>.ExecuteReader(sql);
            sql += " WHERE ID=@ID";

            return DBManager<ORDERED>.ExecuteReader(sql, new { ID = ID });
        }

        public virtual dynamic SelectPaging(int start = 0, int end = 10, string month = "", string cust_id = "", string status = "")
        {
            var sql = string.Format(@"SELECT * FROM(SELECT ROW_NUMBER() OVER (order by ORDED_DATE) AS ROWNUM, O.*,CUS.NAME,E.EMP_NAME,
(SELECT TOP 1 NAME FROM END_USER WHERE CUS_ID=O.END_USER) as END_USER_NAME
FROM ORDERED AS O
INNER JOIN CUSTOMER AS CUS ON O.CUSTOMER_ID=CUS.ID INNER JOIN EMPLOYEE AS E ON E.EMP_ID=O.EMP_ID
WHERE (@month='' OR ORDED_DATE LIKE @month+'%')
            AND (@cust_id='' OR CUSTOMER_ID=@cust_id) AND(@status='' OR ORD_STAT=@status)
) as u  WHERE   RowNum >= @start   AND RowNum < @end ORDER BY RowNum;");

            return DBManager<ORDERED>.ExecuteDynamic(sql, new
            {
                start = start,
                end = end,
                month = month,
                cust_id = cust_id,
                status = status
            });
        }
        public  dynamic GetExport( string month = "", string cust_id = "", string status = "")
        {
            var sql = string.Format(@"SELECT O.*,CUS.NAME,E.EMP_NAME,
(SELECT TOP 1 NAME FROM END_USER WHERE CUS_ID=O.END_USER) as END_USER_NAME
FROM ORDERED AS O
INNER JOIN CUSTOMER AS CUS ON O.CUSTOMER_ID=CUS.ID INNER JOIN EMPLOYEE AS E ON E.EMP_ID=O.EMP_ID
WHERE (@month='' OR ORDED_DATE LIKE @month+'%')
            AND (@cust_id='' OR CUSTOMER_ID=@cust_id) AND(@status='' OR ORD_STAT=@status)
");

            return DBManager<ORDERED>.ExecuteDynamic(sql, new
            {
              
                month = month,
                cust_id = cust_id,
                status = status
            });
        }

        public virtual int GetCount(string month = "", string cust_id = "", string status = "")
        {
            var sql = string.Format(@"SELECT COUNT(1) AS CNT FROM ORDERED WHERE (@month='' OR ORDED_DATE LIKE @month+'%')
            AND (@cust_id='' OR CUSTOMER_ID=@cust_id) AND(@status='' OR ORD_STAT=@status);");
            return (int)DBManager<ORDERED>.ExecuteScalar(sql, new
            {

                month = month,
                cust_id = cust_id,
                status = status
            });
        }

        public virtual int Insert(string EMP_ID, DateTime ORDED_DATE, string CUSTOMER_ID, string ORDER_CR_HR, string STS_ST_CLS, string STS_ST_SER,
            string SURFACE_CD, decimal ORD_THK, decimal ORD_WTH, string ORD_EDGE, decimal ORD_WGT, decimal BASE_PRICE, decimal EFFECT_PRICE,
            decimal BIDD_PRICE, string CONTRACT_NO, string ORD_USAGE, string ORD_STAT, string END_USER, int QUANTITY, string DELIVERY_TIME, string REMARK)
        {
            var id = this.GenerateId();
            var sql = "INSERT INTO ORDERED(ID,EMP_ID,ORDED_DATE,CUSTOMER_ID,ORDER_CR_HR,STS_ST_CLS,STS_ST_SER,SURFACE_CD,ORD_THK,ORD_WTH,ORD_EDGE,ORD_WGT,BASE_PRICE,EFFECT_PRICE,BIDD_PRICE,CONTRACT_NO,ORD_USAGE,ORD_STAT,END_USER,QUANTITY,DELIVERY_TIME,REMARK) VALUES(@ID,@EMP_ID,@ORDED_DATE,@CUSTOMER_ID,@ORDER_CR_HR,@STS_ST_CLS,@STS_ST_SER,@SURFACE_CD,@ORD_THK,@ORD_WTH,@ORD_EDGE,@ORD_WGT,@BASE_PRICE,@EFFECT_PRICE,@BIDD_PRICE,@CONTRACT_NO,@ORD_USAGE,@ORD_STAT,@END_USER,@QUANTITY,@DELIVERY_TIME,@REMARK)";
            return DBManager<ORDERED>.Execute(sql, new
            {
                ID = id,
                EMP_ID = EMP_ID,
                ORDED_DATE = ORDED_DATE,
                CUSTOMER_ID = CUSTOMER_ID,
                ORDER_CR_HR = ORDER_CR_HR,
                STS_ST_CLS = STS_ST_CLS,
                STS_ST_SER = STS_ST_SER,
                SURFACE_CD = SURFACE_CD,
                ORD_THK = ORD_THK,
                ORD_WTH = ORD_WTH,
                ORD_EDGE = ORD_EDGE,
                ORD_WGT = ORD_WGT,
                BASE_PRICE = BASE_PRICE,
                EFFECT_PRICE = EFFECT_PRICE,
                BIDD_PRICE = BIDD_PRICE,
                CONTRACT_NO = CONTRACT_NO,
                ORD_USAGE = ORD_USAGE,
                ORD_STAT = ORD_STAT,
                END_USER = END_USER,
                QUANTITY=QUANTITY,
                DELIVERY_TIME=DELIVERY_TIME,
                REMARK=REMARK
            });
        }

        public virtual int Update(string ID, string EMP_ID, DateTime ORDED_DATE, string CUSTOMER_ID, string ORDER_CR_HR, string STS_ST_CLS, string STS_ST_SER, string SURFACE_CD, decimal ORD_THK, decimal ORD_WTH, string ORD_EDGE, decimal ORD_WGT, decimal BASE_PRICE, decimal EFFECT_PRICE, decimal BIDD_PRICE, string CONTRACT_NO, string ORD_USAGE, string ORD_STAT, string END_USER, int QUANTITY,string DELIVERY_TIME, string REMARK)
        {
            var sql = "UPDATE ORDERED SET EMP_ID=@EMP_ID, ORDED_DATE=@ORDED_DATE,CUSTOMER_ID=@CUSTOMER_ID,ORDER_CR_HR=@ORDER_CR_HR,STS_ST_CLS=@STS_ST_CLS,STS_ST_SER=@STS_ST_SER,SURFACE_CD=@SURFACE_CD,ORD_THK=@ORD_THK,ORD_WTH=@ORD_WTH,ORD_EDGE=@ORD_EDGE,ORD_WGT=@ORD_WGT,BASE_PRICE=@BASE_PRICE,EFFECT_PRICE=@EFFECT_PRICE,BIDD_PRICE=@BIDD_PRICE,CONTRACT_NO=@CONTRACT_NO,ORD_USAGE=@ORD_USAGE,ORD_STAT=@ORD_STAT,END_USER=@END_USER,QUANTITY=@QUANTITY,DELIVERY_TIME=@DELIVERY_TIME,REMARK=@REMARK WHERE ID=@ID";

            return DBManager<ORDERED>.Execute(sql, new
            {
                ID = ID,
                EMP_ID = EMP_ID,
                ORDED_DATE = ORDED_DATE,
                CUSTOMER_ID = CUSTOMER_ID,
                ORDER_CR_HR = ORDER_CR_HR,
                STS_ST_CLS = STS_ST_CLS,
                STS_ST_SER = STS_ST_SER,
                SURFACE_CD = SURFACE_CD,
                ORD_THK = ORD_THK,
                ORD_WTH = ORD_WTH,
                ORD_EDGE = ORD_EDGE,
                ORD_WGT = ORD_WGT,
                BASE_PRICE = BASE_PRICE,
                EFFECT_PRICE = EFFECT_PRICE,
                BIDD_PRICE = BIDD_PRICE,
                CONTRACT_NO = CONTRACT_NO,
                ORD_USAGE = ORD_USAGE,
                ORD_STAT = ORD_STAT,
                QUANTITY = QUANTITY,
                END_USER=END_USER,
                REMARK = REMARK,
                DELIVERY_TIME=DELIVERY_TIME
            });
        }

        public virtual int Delete(string ID = "")
        {
            var sql = "DELETE FROM ORDERED ";
            if (string.IsNullOrWhiteSpace(ID)) return DBManager<ORDERED>.Execute(sql);
            sql += " WHERE ID=@ID ";
            return DBManager<ORDERED>.Execute(sql, new { ID = ID });
        }

        public string GenerateId()
        {
            string id = DateTime.Now.ToString("yyMMdd");
            string sql = "select top 1 ID  from ORDERED WHERE ID LIKE @id + '%' order by ID desc";

            ORDERED claim = DBManager<ORDERED>.ExecuteReader(sql, new { id = id }).FirstOrDefault();
            if (claim == null)
            {
                id = id + "0001";
            }
            else
            {
                string str = claim.ID.Trim().Substring(6);
                string num = (Convert.ToInt32(str) + 1).ToString();
                for (int i = 0; i < str.Length - num.Length; i++)
                {
                    id += "0";
                }
                id += num.ToString();
            }
            return id;
        }
    }

}

File: VstCustomer/Scripts/order.js

$(document).ready(function () {
    $('#loder').addClass('active');
    $('#from,#to')
    .datepicker({
        format: 'yyyy-mm-dd'
    }).on('changeDate', function (ev) {
        $(this).datepicker('hide');
    });
    $('#txtDelivery')
     .datepicker({
         format: 'yyyy-mm',
         viewMode: 'months'
     }).on('changeDate', function (ev) {
         $(this).datepicker('hide');
     });
    $('#txtOrderDate')
    .datepicker({
        format: 'yyyy-mm-dd'
    }).on('changeDate', function (ev) {
        $(this).datepicker('hide');
    });
    $('#btnRegOrder').click(function () {
        $("h4").html("<span class='glyphicon glyphicon-edit'></span> NEW ORDER");
        $('#frmRegOrder')[0].reset();
        $('#txtEmpId').val($('#username').val());
        $('#txtEmpName').val($('#username').attr('data-name'));
        $('#frmRegOrder').attr('data-action', 0);
        $("#mdRegOrder").modal({
            backdrop: 'static',
            keyboard: false
        });
        return false;
    });
    $("#tbMainDefault").on('click', 'tr', function (e) {
        if (!$(this).parent("thead").is('thead')) {
            $("tr").removeClass("success");
            $(this).addClass("success");
            if (!$(e.target).is('#tbMainDefault td input:checkbox')) {
                $(this).find('input:checkbox').trigger('click');
            }
        }
    });
    $("#tbMainDefault").on('change', '.ckb', function () {
        var group = ":checkbox[class='" + $(this).attr("class") + "']";
        if ($(this).is(':checked')) {
            $(group).not($(this)).attr("checked", false);

        }

    });
   
    var tbClaim = $('#tbMainDefault').DataTable(
         {
             sort: false,
             "processing": true,
             "serverSide": true,
             "searching": false,
             ajax: {
                 type: "POST",
                 contentType: "application/json",
                 url: $('#hdUrl').val().replace("Action", "GetOrder"),
                 data: function (d) {
                     // note: d is created by datatable, the structure of d is the same with DataTableParameters model above

                     return JSON.stringify({
                         dataTableParameters: d,
                         from: $('#from').val(),
                         to: $('#to').val(),
                         cust_id: $('#selCus').val(),
                         status: $('#selStatus').val(),
                     });
                 }
             }

         });
    $('#btnDel').click(function () {
        if ($("input:checkbox:checked").length > 0) {
            var id = $("#tbMainDefault tr").find("input[type='checkbox']:checked").attr('id');
            var id = $("#tbMainDefault tr").find("input[type='checkbox']:checked").attr('id');
            var em = $("#tbMainDefault tr").find("input[type='checkbox']:checked").attr('data-emp');
            var role = $('#username').attr('data-role');
            var emp = $('#username').val();
            if (em != emp && role < 1) {
                bootbox.alert('You cannot delete this order');
                return false;
            }
            var cfm = confirm("Are you sure you want to delete this claim?");
            if (cfm) {
                $.ajax({
                    url: $('#hdUrl').val().replace("Action", "Delete"),
                    data: JSON.stringify({
                        ID: id
                    }),
                    type: 'POST',
                    dataType: 'json',
                    contentType: 'application/json; charset=utf-8',
                    crossBrowser: true,
                    success: function (data, status) {
                        if (data > 0) {
                            tbClaim.ajax.reload();
                            bootbox.alert(status);
                        }
                        else {
                            bootbox.alert('Delete fail');
                        }
                        return false;
                    },
                    error: function (xhr, status, error) {
                        bootbox.alert("Error! " + xhr.status);
                    },
                });
            }
        }
        else {
            bootbox.alert("Please select a row");
        }
        return false;
    });
    $("#btnCopy").click(function () {
        if ($("input:checkbox:checked").length > 0) {
            $('#frmRegOrder')[0].reset();
            $("h4").html("<span class='glyphicon glyphicon-edit'></span> NEW ORDER");
            var id = $("#tbMainDefault tr").find("input[type='checkbox']:checked").attr('id');
            var cus_id = $("#tbMainDefault tr").find("input[type='checkbox']:checked").attr('data-cus');
            var emp_id = $('#username').val();
            $('#frmRegOrder').attr('data-id', '');
            $('#frmRegOrder').attr('data-action', 0);
           
            if (id) {
                $.ajax({
                    url: $('#hdUrl').val().replace("Action", "GetOrderById"),
                    data: JSON.stringify({
                        ID: id
                    }),
                    type: 'POST',
                    dataType: 'json',
                    contentType: 'application/json; charset=utf-8',
                    crossBrowser: true,
                    success: function (data, status) {
                        if (data.length > 0) {
                            for (var i = 0; i < data.length; i++) {
                                var ORDED_DATE = data[i].ORDED_DATE == null ? "" : moment(data[i].ORDED_DATE).format('YYYY-MM-DD');
                                $('#txtOrderDate').val(ORDED_DATE);
                                $('#txtEmpId').val($.trim(emp_id));
                                $('#selCr').val($.trim(data[i].ORDER_CR_HR));
                                $('#selCustomer').val($.trim(data[i].CUSTOMER_ID));
                                $('#selCustomer').change();
                                $('#selSurface').val($.trim(data[i].SURFACE_CD));
                                $('#selTHK').val(data[i].ORD_THK);
                                $('#selWTH').val(data[i].ORD_WTH);
                                $('#selEdge').val(data[i].ORD_EDGE);
                                //$('#selWGT').val(data[i].ORD_WGT);
                                $('#txtBasePrice').val(data[i].BASE_PRICE);
                                $('#txtEffPrice').val(data[i].EFFECT_PRICE);
                                $('#txtBidPrice').val(data[i].BIDD_PRICE);
                                $('#txtOrderNo').val(data[i].CONTRACT_NO);
                                $('#selUsage').val(data[i].ORD_USAGE);
                                $('#selStt').val($.trim(data[i].ORD_STAT));
                                $('#txtQuantity').val($.trim(data[i].QUANTITY));
                                $('#selEndUser').val($.trim(data[i].END_USER));
                                $('#selGrade').val($.trim(data[i].STS_ST_CLS));
                                $('#selSeries').val($.trim(data[i].STS_ST_SER));
                                $('#selGrade').val($.trim(data[i].STS_ST_CLS));
                                $('#txtDelivery').val($.trim(data[i].DELIVERY_TIME));
                                $('#txtRemark').val($.trim(data[i].REMARK));
                                $('#selEndUser').change();
                            }
                        }

                        return false;
                    },
                    error: function (xhr, status, error) {
                        bootbox.alert("Error! " + xhr.status);
                    },
                });
              
            }
            $("#mdRegOrder").modal({
                backdrop: 'static',
                keyboard: false
            });
        }
        else {
            bootbox.alert("Please select a row.");
        }
        return false;
    });
    $("#btnModify").click(function () {
        if ($("input:checkbox:checked").length > 0) {
            $("h4").html("<span class='glyphicon glyphicon-edit'></span> MODIFY ORDER");
            var id = $("#tbMainDefault tr").find("input[type='checkbox']:checked").attr('id');
            var cus_id = $("#tbMainDefault tr").find("input[type='checkbox']:checked").attr('data-cus');
            var emp_id = $("#tbMainDefault tr").find("input[type='checkbox']:checked").attr('data-emp');
            $('#frmRegOrder').attr('data-id', id);
            $('#frmRegOrder').attr('data-action', 1);
            $('#frmRegOrder')[0].reset();
            if (id) {
                $.ajax({
                    url: $('#hdUrl').val().replace("Action", "GetOrderById"),
                    data: JSON.stringify({
                        ID: id
                    }),
                    type: 'POST',
                    dataType: 'json',
                    contentType: 'application/json; charset=utf-8',
                    crossBrowser: true,
                    success: function (data, status) {
                        if (data.length > 0) {
                            for (var i = 0; i < data.length; i++) {
                                var ORDED_DATE = data[i].ORDED_DATE == null ? "" : moment(data[i].ORDED_DATE).format('YYYY-MM-DD');
                                $('#txtOrderDate').val(ORDED_DATE);
                                $('#txtEmpId').val($.trim(emp_id));
                                $('#selCr').val($.trim(data[i].ORDER_CR_HR));
                                $('#selCustomer').val($.trim(data[i].CUSTOMER_ID));
                                //$('#selEndUser').val();
                               
                                $('#selCustomer').change();
                                $('#selSurface').val($.trim(data[i].SURFACE_CD));
                                $('#selTHK').val(data[i].ORD_THK);
                                $('#selWTH').val(data[i].ORD_WTH);
                                $('#selEdge').val(data[i].ORD_EDGE);
                                //$('#selWGT').val(data[i].ORD_WGT);
                                $('#txtBasePrice').val(data[i].BASE_PRICE);
                                $('#txtEffPrice').val(data[i].EFFECT_PRICE);
                                $('#txtBidPrice').val(data[i].BIDD_PRICE);
                                $('#txtOrderNo').val(data[i].CONTRACT_NO);
                                $('#selUsage').val(data[i].ORD_USAGE);
                                $('#selStt').val($.trim(data[i].ORD_STAT));
                                $('#txtQuantity').val($.trim(data[i].QUANTITY));
                                $('#selEndUser').val($.trim(data[i].END_USER));
                                $('#selGrade').val($.trim(data[i].STS_ST_CLS));
                                $('#selSeries').val($.trim(data[i].STS_ST_SER));
                                $("#selEndUser option:selected").attr($.trim(data[i].END_USER));
                                $('#txtDelivery').val($.trim(data[i].DELIVERY_TIME));
                                $('#txtRemark').val($.trim(data[i].REMARK));
                                $('#selEndUser').change();
                            }
                        }

                        return false;
                    },
                    error: function (xhr, status, error) {
                        bootbox.alert("Error! " + xhr.status);
                    },
                });
                
            }
            $("#mdRegOrder").modal({
                backdrop: 'static',
                keyboard: false
            });
        }
        else {
            bootbox.alert("Please select a row.");
        }
        return false;
    });
   
    $('#selCustomer').change(function () {
        $("#cus2 option").remove();
        var cus_id =$.trim( $(this).val());
        if ($(this).val().length >= 5) {
            $.ajax({
                url: $('#hdUrl').val().replace("Action", "GetEndUser"),
                data: JSON.stringify({
                    CUS_ID: cus_id
                }),
                type: 'POST',
                dataType: 'json',
                contentType: 'application/json; charset=utf-8',
                crossBrowser: true,
                success: function (data, status) {
                    var e = data.END;
                    $('#selEndUser,#txtEndUser').val('');
                 
                    if ($.trim(e[0].END_USER_ID) == cus_id) {
                        $('#selEndUser').val($.trim(cus_id));
                        $('#selEndUser').change();
                    }
                   
                    $('#txtCustName').val($.trim(data.CUSTOMER.NAME));
                    for (var i = 0; i < e.length; i++) {
                       
                        $('#cus2').append("<option value='" + $.trim(e[i].END_USER_ID) + "'>");
                      
                    }

                    return false;
                },
                error: function (xhr, status, error) {
                    bootbox.alert("Error! " + xhr.status);
                },
            });
        }
        return false;
    });
    $('#selEndUser').change(function () {
      
        var cus_id = $.trim($(this).val());
        if ($(this).val().length >= 5) {
            $.ajax({
                url: $('#hdUrl').val().replace("Action", "GetEndUserById"),
                data: JSON.stringify({
                    ID: cus_id
                }),
                type: 'POST',
                dataType: 'json',
                contentType: 'application/json; charset=utf-8',
                crossBrowser: true,
                success: function (data, status) {
                    $('#txtEndUser').val($.trim(data.NAME));
                    $('#selEndUser').val($.trim(cus_id));
                    return false;
                },
                error: function (xhr, status, error) {
                    bootbox.alert("Error! " + xhr.status);
                },
            });
        }
        return false;
    });
    $('#frmRegOrder').submit(function (e) {
        e.preventDefault();
        //if ($('#frmRegOrder').smkValidate()) {
        var id = $.trim($('#frmRegOrder').attr('data-id'));
        var action = $('#frmRegOrder').attr('data-action');

        var order = {
            ID: id,
            ORDED_DATE: $.trim($('#txtOrderDate').val()),
            EMP_ID: $.trim($('#txtEmpId').val()),
            ORDER_CR_HR: $.trim($('#selCr').val()),
            CUSTOMER_ID: $.trim($('#selCustomer').val()),
            SURFACE_CD: $.trim($('#selSurface').val()),
            ORD_THK: $.trim($('#selTHK').val()),
            ORD_WTH: $.trim($('#selWTH').val()),
            ORD_EDGE: $.trim($('#selEdge').val()),
            ORD_WGT: 0, //$.trim($('#selWGT').val()),
            BASE_PRICE: $.trim($('#txtBasePrice').val()),
            EFFECT_PRICE: $.trim($('#txtEffPrice').val()),
            BIDD_PRICE: $.trim($('#txtBidPrice').val()),
            CONTRACT_NO: $.trim($('#txtOrderNo').val()),
            ORD_USAGE: $.trim($('#selUsage').val()),
            ORD_STAT: $.trim($('#selStt').val()),
            STS_ST_CLS: $.trim($('#selGrade').val()),
            STS_ST_SER: $.trim($('#selSeries').val()),
            QUANTITY: $('#txtQuantity').val(),
            END_USER: $('#selEndUser').val(), 
            DELIVERY_TIME: $.trim($('#txtDelivery').val()),
            REMARK: $.trim($('#txtRemark').val())
        };


        $.ajax({
            url: $('#hdUrl').val().replace("Action", "InsertUpdateOrder"),
            data: JSON.stringify({
                ORDER: order,
                ACTION: action
            }),
            type: 'POST',
            dataType: 'json',
            contentType: 'application/json; charset=utf-8',
            crossBrowser: true,
            success: function (data, status) {
                bootbox.alert(status);
                tbClaim.ajax.reload();
                //$('#mdRegOrder').modal('hide');
                $('#frmRegOrder')[0].reset();
                return false;
            },
            error: function (xhr, status, error) {
                bootbox.alert("Error! " + xhr.status);
            },
        });

        //}
        //else {
        //    bootbox.alert('Invalid field');
        //}

        return false;
    });
    
    $('#btnExport').click(function () {
        var from = $('#from').val();
        var to = $('#to').val();
        var cus_id = $('#selCus').val();
        var status = $('#selStatus').val();
        var url = $('#hdUrl').val().replace("Action", "Export") + "?from=" + from + "&to="+to+"&cus_id=" + cus_id + "&status=" + status;
        window.location.href = url;
        return false;
    });
    $('#btnExportReport').click(function () {
        var from = $('#from').val();
        var to = $('#to').val();
        var cus_id = $('#selCus').val();
        var status = $('#selStatus').val();
        var url = $('#hdUrl').val().replace("Action", "ExportReport") + "?from=" + from + "&to=" + to + "&cus_id=" + cus_id + "&status=" + status;
        window.location.href = url;
        return false;
    });
    $('#btnSearch').click(function () {
        tbClaim.draw();
        return false;
    });
});